client/dashboard/dashboard-charts/index.js

```diff
@@ -76,21 +77,27 @@ class DashboardCharts extends Component {
 		const { hiddenChartKeys } = this.state;
 
 		return (
-			<EllipsisMenu label={ __( 'Choose which charts to display', 'woocommerce-admin' ) }>
-				{ uniqCharts.map( chart => {
-					return (
-						<MenuItem
-							checked={ ! hiddenChartKeys.includes( chart.key ) }
-							isCheckbox
-							isClickable
-							key={ chart.key }
-							onInvoke={ this.toggle( chart.key ) }
-						>
-							{ __( `${ chart.label }`, 'woocommerce-admin' ) }
-						</MenuItem>
-					);
-				} ) }
-			</EllipsisMenu>
+			<EllipsisMenu
+				label={ __( 'Choose which charts to display', 'woocommerce-admin' ) }
+				renderChildren={ ( { onToggle } ) => (
+					<Fragment>
+						{ uniqCharts.map( chart => {
+							return (
+								<MenuItem
+									checked={ ! hiddenChartKeys.includes( chart.key ) }
+									isCheckbox
+									isClickable
+									key={ chart.key }
+									onInvoke={ this.toggle( chart.key ) }
+								>
+									{ __( `${ chart.label }`, 'woocommerce-admin' ) }
+								</MenuItem>
+							);
+						} ) }
+						<SectionControls onToggle={ onToggle } { ...this.props } />
```

Comment:
I'm a bit afraid passing down all props (`{ ...this.props }`) to the `SectionControls` will become messy in the future as we keep adding new props to each section. How about declaring them one-by-one? It makes code longer and more verbose but I think it will help to understand which props are required by `SectionControls` and which don't.